backend/tests/reservations.test.js

```diff
@@ -0,0 +1,58 @@
+const db = require('../src/db');
+const utils = require('../src/utils');
+
+describe('Reservation Testing', () => {
+    let reservation = {
+        id: 1,
+        courseId: 186399,
+        eventId: 2554743,
+        userId: 123,
+        ticketType: 'Single_Ticket',
+        ticketPrice: 90.5,
+        bookingStatus: 0,
+    };
+
+    test('Should create reservations', async () => {
+        await db.reservations.createReservation(reservation);
+        const dbreservation = await db.reservations.getReservationById(
+            reservation.id
+        );
+        expect(dbreservation.id).toEqual(reservation.id);
```

Comment:
Shouldn't the reservation's id be coming from the database instead of being set by the parameters?